mkdocs.yml

```diff
@@ -6,8 +6,19 @@ theme:
   name: material
   custom_dir: docs/overrides
   palette:
-    primary: black
-    accent: teal
+    - media: "(prefers-color-scheme: light)"
+      primary: black
+      accent: teal
+      toggle:
+        icon: material/brightness-7
```

Comment:
I would consider matching these icons to the FastApi [documentation](https://github.com/tiangolo/fastapi/blob/6879082b3668edd213d035b6e9a90a4bccf32e01/docs/en/mkdocs.yml#L13) to maintain consistency between the projects.

Reply:
good idea